src/main/java/org/folio/inventory/domain/ingest/IngestMessageProcessor.java

```diff
@@ -146,7 +147,9 @@ private void processRecordsMessage(Message<JsonObject> message, final EventBus e
                 ? loanTypes.getString("Can Circulate")
                 : loanTypes.getString("Can circulate"),
                null)
-                    .withBarcode(record.getString("barcode"));
+                    .withBarcode(record.getString("barcode"))
+                .withLastCheckIn(
```

Comment:
Why would an ingested record ever have a last check in?

Reply:
It was added as a `safe` measure... It will not break/change anything. Do you think it should be removed?

Reply:
Removed.